devfile.yaml

```diff
@@ -33,17 +33,51 @@ components:
 
 commands:
 
-  - name: yarn
+  - name: Build
     actions:
       - type: exec
         component: dev
-        command: yarn
+        command: >
+                 yarn
         workdir: /projects/chectl
 
-  - name: yarn test
+  - name: Test
     actions:
       - type: exec
         component: dev
-        command: yarn test
+        command: >
+                 yarn test
         workdir: /projects/chectl
 
+  - name: Run
+    actions:
+      - type: exec
+        component: dev
+        command: >
+                 ./run --help
+        workdir: /projects/chectl/bin
+
+  - name: Install Packager
+    actions:
+      - type: exec
+        component: dev
+        command: >
+                 npm install -g pkg
+        workdir: /projects/chectl
+
+  - name: Package Binaries
```

Comment:
just to know, it's the deprecated binaries

it's `oclif-dev pack` now

Reply:
Thanks. I will try

Reply:
`oclif-dev pack` does not create executable file. It creates a package, which contains `node` with a set of javascript files.

Reply:
Reworked